packages/x-charts/src/context/PolarProvider/PolarProvider.tsx

```diff
@@ -1,59 +1,60 @@
 'use client';
-import * as React from 'react';
-import { computeAxisValue } from '../../internals/computeAxisValue';
-import { useDrawingArea } from '../../hooks/useDrawingArea';
-import { useSeries } from '../../hooks/useSeries';
-import { PolarContext } from './PolarContext';
-import { useRadiusExtremumGetter } from '../PluginProvider/useRadiusExtremumGetter';
-import { useRotationExtremumGetter } from '../PluginProvider/useRotationExtremumGetter';
-import { PolarProviderProps } from './Polar.types';
+// 'use client';
+// import * as React from 'react';
+// import { computeAxisValue } from '../../internals/plugins/featurePlugins/useChartCartesianAxis/computeAxisValue';
```

Comment:
I just commented the file about polar coordinate. They got introduced when starting the effort on radar chart.

They should be translated into a `useChartPolarAxisPlugin`